src/components/Schedule/NewScheduleForm.tsx

```diff
@@ -77,9 +77,13 @@ export const NewScheduleForm = ({ onClose, error, onSubmit }: NewScheduleFormPro
 
   return (
     <div className={classes.root}>
-      <h2>New Schedule Form</h2>
+      <h2>Create a new schedule</h2>
```

Comment:
```suggestion
      <h2>Create A New Schedule</h2>
```
Capitalize the `<h2>`, to be consistent with NewServiceForm.js's "Add A New Event"